RELEASE.md

```diff
@@ -1,3 +1,83 @@
+# Release 1.15.0
+
+## Major Features and Improvements
+
+## Breaking Changes
+
+## Bug Fixes and Other Changes
+
+* Promoting `unbatch` from experimental to core API.
+* Adds `tf.enable_control_flow_v2()` and `tf.disable_control_flow_v2()`.
+* EagerTensor now support buffer interface for tensors.
```

Comment:
If we add an example that makes clear what that means, we should move this to major features.

Reply:
Done. Checked with alextp@ and added "numpy buffer interface".